inference/tests/test_app.py

```diff
@@ -5,4 +5,6 @@
 def test_uploadfile():
     client = app.test_client()
     response = client.post("/upload_file", data=json.dumps(dict(f='f')))
-    assert response.status_code == 200
+    assert response.status_code == 400
```

Comment:
why remove the success 200 case?

Reply:
We are given .json files as input

Reply:
please create a success test case and a failure one.

Reply:
The addition can soon be found in #35